core/src/main/scala/stryker4s/mutants/findmutants/FileCollector.scala

```diff
@@ -67,6 +67,7 @@ class FileCollector(implicit config: Config) extends SourceCollector with Loggin
   private[this] def listAllFiles(): Option[Iterable[File]] = {
     Option(config.baseDir
       .listRecursively
+      .filterNot(file => file.isDirectory)
```

Comment:
Oh woops, didn't mean for this to be here.

This line of code fixes a problem I had for my test project. Because I did not use a git repo the FileCollector chose to copy the whole directory. But the directories are part of this list too. 

So then what happens is an empty file '.idea' is created (which should be a directory). And then it tries to created a directory '.idea' to put a file in and I get `java.nio.file.FileAlreadyExistsException: C:\path\to\stryker\target\stryker4s-6944695956008966908\.idea`.

Reply:
This is already fixed in the `sbt-plugin` branch. Could you revert it to prevent a merge conflict? 😀